### DISCUSSION TITLE
Powershell 7 not installing modules into correct module path?

### DISCUSSION THREAD

**Discussion by mcdonamw** [PowerShell/PowerShell#21272] [Q&A] [answered]
I just noticed that when I install modules using Powershell 7, they are going into the C:\Program Files\Powershell\Modules path, not C:\Program Files\Powershell\7\Modules.  Is this expected?  I thought 7 installed into the path with 7 in the name?

If expected, what's the difference between those two paths?

**Comment by rhubarb-geek-nz** [comment, marked as answer]
If you look at PSModulePath, that might help

```
PS> $env:PSModulePath
D:\OneDrive\Documents\PowerShell\Modules;C:\Program Files\PowerShell\Modules;c:\program files\powershell\7\Modules;C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules
```
| path | comment |
| ---- | ----- |
| D:\OneDrive\Documents\PowerShell\Modules | powershell core modules installed in user scope |
| C:\Program Files\PowerShell\Modules | powershell core modules installled system wide |
| c:\program files\powershell\7\Modules | prepackaged modules that came in the original PowerShell core MSI installation |
| C:\Program Files\WindowsPowerShell\Modules | Windows PowerShell modules, for PowerShell 5.1 |
| C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules | prepackaged modules that come with Windows PowerShell 5.1 |

If you run PowerShell 5.1 you get

```
PS> $env:PSModulePath
D:\OneDrive\Documents\WindowsPowerShell\Modules;C:\Program Files\WindowsPowerShell\Modules;C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules
```

| path | comment |
| ----- | ---------- |
| D:\OneDrive\Documents\WindowsPowerShell\Modules | user scope modules for Windows PowerShell 5.1 |
| C:\Program Files\WindowsPowerShell\Modules | system scope installed modules for Windows PowerShell 5.1 |
| C:\WINDOWS\system32\WindowsPowerShell\v1.0\Modules | prepackaged Windows PowerShell modules 5.1 |

**Comment by kilasuit** [comment]
Yes this is as expected as explained above by @rhubarb-geek-nz
